### PR TITLE
Use 'the library' instead of the gendered pronoun 'he' in the Readme file

### PR DESCRIPTION
**Pull request by jblz**
This is, of course, a matter of some linguistic preference, but seeing as the word `one` is used earlier in the sentence, it's already set up to be gender-neutral :)

Some food for thought:
* http://en.wikipedia.org/wiki/Gender-specific_and_gender-neutral_pronouns#Singular_they
* http://www.oxforddictionaries.com/words/he-or-she-versus-they

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ var colorCursor = paletteCursor.select('colors');
 
 #### Updates
 
-A *baobab* tree can obviously be updated. However, one has to understand that he won't do it, at least by default, synchronously.
+A *baobab* tree can obviously be updated. However, one has to understand that they won't do it, at least by default, synchronously.
```
Comment (jblz) [inline README.md]: `he or she` would accomplish the same goal, but I prefer the singular `they`, myself.

Comment (Yomguithereal) [inline README.md]: Shouldn't we use `it`? I am not sure whether this is the same for English as for my native language but I am referring to the Baobab tree in this sentence, not the user.

Comment (SebastienDaniel) [inline README.md]: The proper formulation is:
> A *baobab* tree can obviously be updated. However, one has to understand that **it** won't do **so**, at least by default, synchronously.

Comment (jblz) [inline README.md]: Oh, yes, sorry. I misinterpreted this line as pertaining to the user.

I would be explicit here, then, and say:
> **the library** won't do **so**

**Comment by Yomguithereal** [comment]
Merged :).

**Pull request merged**